helm/hpcc/templates/_helpers.tpl

```diff
@@ -1334,18 +1334,23 @@ Pass in dict with root, job, target and type
 {{- $job := .job -}}
 {{- $target := (printf "target:%s" .target | default "") -}}
 {{- $type := printf "type:%s" .type -}}
+{{- $categories := list "all" $type $target -}}
 {{- $placementsDict := dict -}}
-{{- range $placement := .root.Values.placements -}}
-{{- if or (has $target $placement.pods) (has $type $placement.pods) (has "all" $placement.pods) -}}
+{{- $placements := .root.Values.placements -}}
+{{- range $category := $categories -}}
+{{- range $placement := $placements -}}
```

Comment:
not entirely new, but it is clearer/easy to read if there is block indentation in these template definitions.

Reply:
Done